docs/content/getting-started/quick-start-with-kubernetes.md

```diff
@@ -1,23 +1,23 @@
 ---
 title: "Traefik Getting Started With Kubernetes"
-description: "Looking to get started with Traefik Proxy? Read the technical documentation to learn a simple use case that leverages Kubernetes."
+description: "Get started with Traefik Proxy and Kubernetes."
```

Comment:
I am not sure that we should update the description because this was introduced to improve the SEO by @tomatokoolaid.